.gitignore

```diff
@@ -1 +1,2 @@
-node_modules
+node_modules/
+/lib/
```

Comment:
See 0ca7cc9aebee196f9e9f81dc32e9d4afa29df0e1 for context and explanation. The directory is removed from git, but included in the published package, and the `prepublish` script will ensure that it's up to date when you publish. If you want to give this a dry run, make `npm pack` and you'll get a tarball of everything that will get published.